apps/api/v2/src/ee/bookings/outputs/mark-no-show.output.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsString, IsEnum, IsOptional, ValidateNested, IsArray, IsEmail, IsBoolean } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+class Attendee {
+  @IsEmail()
+  email!: string;
+
+  @IsBoolean()
+  noShow!: boolean;
+}
+
+class HandleMarkNoShowData {
+  @IsString()
+  message!: string;
```

Comment:
`message` is for frontend. I think it won't make sense for API. It is translation key anyway.

![image](https://github.com/calcom/cal.com/assets/1780212/df346beb-5c43-4761-9ec2-ed330e4e2f43)